provision/vvv-init.sh

```diff
@@ -54,6 +55,31 @@ fi
 cp -f "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf.tmpl" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 sed -i "s#{{DOMAINS_HERE}}#${DOMAINS}#" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 
+
+# Proxy images from live site
+if [[ "${LIVE_URL}" != http* ]]; then
+	LIVE_URL=''
+else
+	LIVE_URL=$(cat <<'END_HEREDOC'
+
+
+	# Directives to send expires headers and turn off 404 error logging.
+	location ~* .(js|css|png|jpg|jpeg|gif|ico)$ {
```

Comment:
My understanding is that this would also match these files in plugins and theme folders too

Reply:
as I can see yes, we need to check that the files are only inside wp-content/uploads